src/styles/boundaryLoaderStyle/index.ts

```diff
@@ -0,0 +1,13 @@
+import CONST from '../../CONST';
+import OptionAlternateTextPlatformStyles from './types';
+
+const optionAlternateTextPlatformStyles: OptionAlternateTextPlatformStyles = {
```

Comment:
Rename to boundaryLoaderStyles (same as on .android.ts)

Reply:
👍🏻